---
layout: page
permalink: /apps/
title: Computer Applications (Apps)
tags: [computer applications, application, app, console, web, embedded, web, tools, packages, research, table]
modified: 21-04-2024
comments: false
---

This page lists published computer applications, including console, web and window-based applications.<br/>
<br/>


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/ican.webp)](https://github.com/ghattab/ican) </span> | **iCAN** is a console application that enables molecular encoding of any organic compound using the Interpretable Carbon-based Array of Neighborhoods method. It offers interpretable representations tailored for machine learning, outperforms previous methods in peptide and protein classification, and aids in drug discovery and disease diagnosis. [[Publication 1](https://academic.oup.com/nargab/article/5/1/lqac103/6980839), [Publication 2](https://doi.org/10.1016/j.csbj.2024.05.035)]. |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/polardiagrams.webp)](https://github.com/AAnzel/Polar-Diagrams-for-Model-Comparison) </span> | **PolarDiagrams** is an interactive console application that enables the evaluation of complex models across various domains, including biology, medicine, climatology, and machine learning. It addresses the challenge of evaluating models using multiple metrics by providing an integrated visualization approach. [[Publication]](https://www.sciencedirect.com/science/article/pii/S0169260723005096). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/dnasmart.webp)](https://github.com/Sombiri/DNAsmart) </span> | **DNAsmart** is an interactive and visual analytical application that enables the evaluation and ranking of DNA storage encoding approaches based on multiple attributes. It provides an accessible and tractable way to solve comparison, ranking, and validation tasks related to DNA storage encoding. [[Publication]](https://www.sciencedirect.com/science/article/pii/S2001037023000624). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/capt.webp)](https://github.com/ghattab/CAPT) </span> | **CAPT** is an interactive web application that supports users in exploration and validation tasks related to phylogeny-based taxonomy. It bridges the gap between phylogeny and taxonomy by offering two simultaneous views: a phylogenetic tree view and a taxonomic icicle view. [[Publication]](https://www.frontiersin.org/articles/10.3389/fgene.2022.891240). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/movis.webp)](https://github.com/AAnzel/MOVIS) </span> | **MOVIS** is a multimodal omics application that enables the import, embedding, clustering, and interactive visualization of different types of omics data with temporal information. It provides insight into patterns and anomalies in an integrated environment, complementing traditional data exploration approaches. [[Publication]](https://www.sciencedirect.com/science/article/pii/S2001037022000526). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/peptidereactor.webp)](https://peptidereactor.mathematik.uni-marburg.de/) </span> | **PeptideREACToR** is a web and console application for peptide encoding performance in the biomedical domain. It provides a baseline of 48 encoding groups tested on 50 datasets, enabling initial encoding selection, objective comparison of new encodings, and sophisticated encoding optimization in automated machine learning pipelines. [[Publication]](https://academic.oup.com/nargab/article/3/2/lqab039/6281452). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/mosga.webp)](https://mosga.mathematik.uni-marburg.de) </span> | **MOSGA** is a versatile web application that combines genome annotation and comparative genomics capabilities. It provides a user-friendly interface to generate and integrate annotations, analyze results in a genome browser, and enable comparative genomics analyses on high-quality genome assemblies.  [[Publication 1](https://academic.oup.com/bioinformatics/article-abstract/36/22-23/5514/6015104), [Publication 2](https://www.sciencedirect.com/science/article/pii/S2001037021004098)]. |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/modelar.webp)](https://github.com/ghattab/MODELAR) </span> | **modelAR** is a modular and evaluative framework to improve Augmented Reality (AR) visualization for surgical applications. It enables testing different visualization techniques and presets for displaying 3D biomedical data in an AR environment. [[Publication]](https://diglib.eg.org/handle/10.2312/evs20201066). |


| - | - |
| <span style="display: inline-block; width:200px"> [![](/images/apps/seevis.webp)](https://github.com/ghattab/seevis) </span> | **SeeVIS** is a console application that enables automated and qualitative visualization of time-lapse microscopy data. It offers a solution to the lack of visualization alternatives for rapid qualitative characterization of microbial colonies using a space-time cube. [[Publication]](https://academic.oup.com/bioinformatics/article/35/10/1802/5140217). |
