---
layout: page
perma(link): /publications/
title: Publications
tags: [research,publications,scholar,APA]
modified: 22-06-2021
comments: false
---

For a curated view of my work, *cf*. [Research](/research). 

### 2021
* **Hattab, G.**, Hatzipanayioti, A., Klimova, A., Pfeiffer, M., Klausing, P., Breucha, M., ..., & Speidel, S. (2021). Investigating the utility of VR for spatial understanding in surgical planning: Evaluation of head-mounted to desktop display. *Scientific Reports*. (accepted)
* Spänig, S., Mohsen, S., **Hattab, G.**, Hauschild, A. C., & Heider, D. (2021). A large-scale comparative study on peptide encodings for biomedical classification. *NAR genomics and bioinformatics*, 3(2), lqab039. [(link)](https://academic.oup.com/nargab/article/3/2/lqab039/6281452).
* Sperlea, T., Kreuder, N., Beisser, D., **Hattab, G.**, Boenigk, J., & Heider, D. (2021). Quantification of the covariation of lake microbiomes and environmental variables using a machine learning‐based framework. *Molecular Ecology*, 30(9), 2131-2144. [(link)](https://onlinelibrary.wiley.com/doi/10.1111/mec.15872).
* Wagner, D., Heider, D., & **Hattab, G.** (2021). Mushroom data creation, curation, and simulation to support classification tasks. *Scientific reports*, 11(1), 1-12. [(link)](https://www.nature.com/articles/s41598-021-87602-3).
* **Hattab, G.**, Rhyne, T. M., & Heider, D. (2021). Correction: Ten simple rules to colorize biological data visualization. *PLoS Computational Biology*, 17(4), e1008901. [(link)](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008901).
* Hufsky, F., Lamkiewicz, K., Almeida, A., Aouacheria, A., Arighi, C., Bateman, A., ... & Marz, M. (2021). Computational strategies to combat COVID-19: useful tools to accelerate SARS-CoV-2 and coronavirus research. *Briefings in bioinformatics*, 22(2), 642-663. [(link)](https://academic.oup.com/bib/article/22/2/642/5955939).

### 2020
* Martin, R., Hackl, T., **Hattab, G.**, Fischer, M. G., & Heider, D. MOSGA: Modular Open-Source Genome Annotator. *Bioinformatics* (Oxford, England), btaa1003. [(link)](https://academic.oup.com/bioinformatics/article-abstract/36/22-23/5514/6015104).
* **Hattab, G.**, Rhyne, T. M., & Heider, D. (2020). Ten simple rules to colorize biological data visualization. *PLoS Computational Biology*, 16(10). [(link)](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008259).
* Martin, R., Löchel, H. F., Welzel, M., **Hattab, G.**, Hauschild, A. C., & Heider, D. (2020). CORDITE: the curated CORona drug InTERactions database for SARS-CoV-2. *Iscience*, 23(7), 101297. [(link)](https://www.sciencedirect.com/science/article/pii/S2589004220304843).
* **Hattab, G.**, Ahlfeld, T., Klimova, A., Koepp, A., Schuerer, M., & Speidel, S. (2020). Uniaxial compression testing and Cauchy stress modeling to design anatomical silicone replicas. *Scientific Reports*, 10(1), 1-7. [(link)](https://www.nature.com/articles/s41598-020-68886-3).
* **Hattab, G.** (2020). Data from Uniaxial Compression testing and validation scripts for Cauchy stress modeling to design anatomical silicone replicas. *Material Cloud Archive*, 10. [(link)](https://archive.materialscloud.org/record/2020.0019/v2)
* **Hattab, G.**, Riediger, C., Weitz, J., & Speidel, S. (2020). A case study: impact of target surface mesh size and mesh quality on volume-to-surface registration performance in hepatic soft tissue navigation. *International journal of computer assisted radiology and surgery*, 15(8), 1235-1245. [(link)](https://link.springer.com/article/10.1007/s11548-020-02123-0)
* **Hattab, G.**, Arnold, M., Strenger, L., Allan, M., Arsentjeva, D., Gold, O., ... & Speidel, S. (2020). Kidney edge detection in laparoscopic image data for computer-assisted surgery. *International journal of computer assisted radiology and surgery*, 15(3), 379-387. [(link)](https://link.springer.com/article/10.1007/s11548-019-02102-0).
* **Hattab, G.**, Meyer, F., Albrecht, R. D., & Speidel, S. (2020). MODELAR: A MODular and EvaLuative framework to improve surgical Augmented Reality visualization. *The Eurographics Association*, Eurographics & Eurovis 2020. [(link)](https://diglib.eg.org/handle/10.2312/evs20201066).

### 2019
* **Hattab, G.**, & Nattkemper, T. W. (2019). SeeVis—3D space-time cube rendering for visualization of microfluidics image data. *Bioinformatics*, 35(10), 1802-1804. [(link)](https://academic.oup.com/bioinformatics/article/35/10/1802/5140217).

### 2018
* Hattab, G. (2018) Analyzing colony dynamics and visualizing cell diversity in spatiotemporal experiements. *Bielefeld University*. (PhD, Doctor rerum naturalium). [(link)](https://pub.uni-bielefeld.de/record/2919935).
* **Hattab, G.**, Wiesmann, V., Becker, A., Munzner, T., & Nattkemper, T. W. (2018). A novel Methodology for characterizing cell subpopulations in automated Time-lapse Microscopy. *Frontiers in bioengineering and biotechnology*, 6, 17. [(link)](https://www.frontiersin.org/articles/10.3389/fbioe.2018.00017/full).

### 2017
* **Hattab, G.**, Schlüter, J. P., Becker, A., & Nattkemper, T. W. (2017). ViCAR: an adaptive and landmark-free registration of time lapse image data from microfluidics experiments. *Frontiers in genetics*, 8, 69. [(link)](https://www.frontiersin.org/articles/10.3389/fgene.2017.00069/full).

### 2016
* **Hattab, G.**, Brink, B. G., Nattkemper, T. W. (2016). A mnemonic card game for your amino acids. *Information+ Conference.* Emily Carr University of Art+Design, 1, 48. [(link)](https://informationplusconference.com/2016/).

### 2015
* **Hattab, G.**, Warschawski, D. E., Moncoq, K., & Miroux, B. (2015). Escherichia coli as host for membrane protein structure determination: a global analysis. *Scientific reports*, 5(1), 1-10. [(link)](https://www.nature.com/articles/srep12097).
* Schlueter, J. P., McIntosh, M., **Hattab, G.**, Nattkemper, T. W., & Becker, A. (2015). Phase Contrast and Fluorescence Bacterial Time-Lapse Microscopy Image Data. *Bielefeld University*. [(link)](https://pub.uni-bielefeld.de/record/2777409).

### 2014
* **Hattab, G.**, Moncoq, K., Warschawski, D., & Miroux, B. (2014). Escherichia coli as host for membrane protein structure determination: A global analysis. *Biophysical Journal*, 106(2), 46a. [(link)](https://www.cell.com/biophysj/fulltext/S0006-3495(13)01570-1).
* **Hattab, G.** (2014). Pattern recognition in a heat maze based spatial system adapted to Drosophila melanogaster. *Université Paris Diderot Sorbonne*. (Master of Science).
* **Hattab, G.**, Suisse, A. Y., Ilioaia, O., Casiraghi, M., Dezi, M., Warnet, X. L., ... & Miroux, B. (2014). Membrane protein production in Escherichia coli: overview and protocols. Membrane Proteins Production for Structural Analysis, *Springer*, 87-106. [(link)](/documents/Hattab14-2.pdf)

### 2012
* **Hattab, G.** (2012). Establishment of a bibliographic and bioinformatics mining tool to research the over- expression of heterologous membrane proteins. *Institute of Biological Physical Chemistry*. (Bachelor of Science).
